packages/elements-dev-portal/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@stoplight/elements-dev-portal",
+  "version": "1.0.0-alpha.1",
+  "description": "UI components for composing beautiful developer documentation.",
+  "keywords": [],
+  "sideEffects": [
+    "web-components.min.js",
+    "src/web-components/**"
+  ],
+  "homepage": "https://github.com/stoplightio/elements",
+  "bugs": "https://github.com/stoplightio/elements/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/elements"
+  },
+  "license": "Apache-2.0",
+  "main": "",
+  "files": [
+    "**/*"
+  ],
+  "engines": {
+    "node": ">=10"
```

Comment:
@marcelltoth was right. We shouldn't put massive restrictions here, because for the most people just using elements, those restrictions are not necessary.

These are more restrictions of our internal tools.